py/torch_tensorrt/fx/test/converters/aten_op/test_rsub_aten.py

```diff
@@ -0,0 +1,29 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt.fx.tools.common_fx2trt import DispatchTestCase, InputTensorSpec
+
+
+class TestRSubConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ("2d_dim_alpha", (2, 1), 2),
+            ("3d_dim_alpha", (2, 1, 2), 2),
+        ]
+    )
+    def test_rsub(self, _, x, alpha):
+        class rsub(nn.Module):
+            def forward(self, input):
+                return torch.rsub(input, input, alpha=alpha)
```

Comment:
Provide 2 different input tensor to test module, otherwise you already get 0 output.

Reply:
Since it is has an alpha, the output will not be 0 in all cases. But makes sense to create a test case with different inputs.

Reply:
Addressed this. Although I have created the new test case from the class `torch.testing._internal.common_utils.test_case` instead of `DispatchTestCase`, since` DispatchTestCase.run_test` does not seem to have the capability to handle multiple inputs. You could correct me if I am wrong, but I could not find a way to work that out